packages/grid/x-data-grid/src/models/gridIconSlotsComponent.ts

```diff
@@ -110,4 +110,9 @@ export interface GridIconSlotsComponent {
    * @default GridRemoveIcon
    */
   DetailPanelCollapseIcon: React.JSXElementConstructor<any>;
+  /**
+   * Icon displayed for deleting the filter from filter Panel.
+   * @default GridCloseIcon
+   */
+  FilterPanelDeleteIcon: React.JSXElementConstructor<any>;
```

Comment:
We should also add a new entry in the components Prop

Reply:
I'm not sure to understand. For now, icons does not appears in `componentsProps`, only in `components`

And the type of `components` extends the icons one 
https://github.com/alexfauquette/material-ui-x/blob/master/packages/grid/x-data-grid/src/internals/models/gridSlotsComponent.ts#L8:L8